byzcoin/bcadmin/main.go

```diff
@@ -297,6 +297,61 @@ var cmds = cli.Commands{
 					},
 				},
 			},
+			{
+				Name:   "adeferred",
+				Usage:  "add deferred darc that already contains the needed rules. Deferred rules use ORs, _sign and _evolve use ANDs",
+				Action: darcAddDeferred,
+				Flags: []cli.Flag{
+					cli.StringSliceFlag{
+						Name:  "identity, id",
+						Usage: "an identity, multiple use of this param is allowed. If empty it will create a new identity. Each provided identity is checked by the evaluation parser.",
+					},
+					cli.StringFlag{
+						Name:   "bc",
+						EnvVar: "BC",
+						Usage:  "the ByzCoin config to use (required)",
+					},
+					cli.StringFlag{
+						Name:  "sign, signer",
+						Usage: "public key which will sign the DARC spawn request (default: the ledger admin identity)",
+					},
+					cli.StringFlag{
+						Name:  "darc",
+						Usage: "DARC with the right to create a new DARC (default is the admin DARC)",
+					},
+					cli.BoolFlag{
+						Name:  "unrestricted",
+						Usage: "add the invoke:evolve_unrestricted rule",
+					},
+					cli.StringFlag{
+						Name:  "out_id",
+						Usage: "output file for the darc id (optional)",
+					},
+					cli.StringFlag{
+						Name:  "out_key",
+						Usage: "output file for the darc key (optional)",
+					},
+					cli.StringFlag{
+						Name:  "desc",
+						Usage: "the description for the new DARC (default: random)",
+					},
+				},
+			},
+			{
+				Name:   "prule",
```

Comment:
What about using `--dry-run` on the darc creation that will print the darc instead of creating it ?

Reply:
My aim was not to simulate a Darc, but just have a handy way to play with those minimum rules groups, so it can be either directly copy/pasted or further modified.